src/base/pages/Root.vue

```diff
@@ -34,11 +34,12 @@ export default {
 
 <style lang="stylus" scoped>
 .root-splash-second
+  opacity 1;
```

Comment:
it adds a `;` when it reorders them :-1:

Reply:
hmmm... I hoped the "no-trailing-semicolons" rule had fixed it, but that just means no semicolons at the end of a block. The problem is the parser is stylus, but the rules are not designed for stylus....